examples/close_cookies/close_cookies.py

```diff
@@ -0,0 +1,45 @@
+"""This example demonstrates how to close popup windows (like promotion form) with AgentQL."""
+
+import time
+
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# Set the URL to the desired website
+URL = "https://www.amazon.com/"
```

Comment:
do we want to use something lighter for the examples?  i thought we're staying away from amazon

Reply:
Like scrapeme.live? Go for it.